src/modules/ModuleTradeShared/composable.pair-input.ts

```diff
@@ -24,6 +25,42 @@ export function useLocalStorageAddrsOrigin(key: string, isActive?: Ref<boolean>)
   return offable
 }
 
+export function useRouteAddrsOrigin({
+  baseToken,
+  additionalBaseToken,
+  isActive,
+}: {
+  baseToken?: Address
+  additionalBaseToken?: Address
+  isActive?: Ref<boolean>
+}) {
+  const params = {
+    tokenA: useRouteParams('tokenA'),
+    tokenB: useRouteParams('tokenB'),
+  }
+
+  return computed({
```

Comment:
```suggestion
  return computed<null | TokensPair<Address>>({
```

Returning `null` in case when route params don't contain valid tokens makes more sense then returning an empty pair.